spec/api/bookyt/api/bookings_spec.rb

```diff
@@ -197,7 +197,7 @@
 
     it 'removes the booking' do
       delete "/api/bookings/#{booking.id}", params, headers
-      expect(response.body).to eq('')
+      expect(response.body).to eq('204')
```

Comment:
Hm... HTTP status `204` means `No Content`... So the body should really be `''` :smile: 

Looks like we should add `body false` for these endpoints: https://github.com/ruby-grape/grape#sending-raw-or-no-data